global.json

```diff
@@ -14,7 +14,7 @@
     }
   },
   "native-tools": {
-    "cmake": "latest"
+    "cmake": "3.21.0"
```

Comment:
FYI there's a PR to bump to 3.26 in runtime: https://github.com/dotnet/runtime/pull/103088

Reply:
yeah we don't use global.json for native tools